src/server/pkg/collection/collection.go

```diff
@@ -249,6 +254,10 @@ func (c *readWriteCollection) DeleteAll() {
 	c.stm.DelAll(c.prefix)
 }
 
+func (c *readWriteCollection) DeleteAllWithPrefix(prefix string) {
+	c.stm.DelAll(path.Join(c.prefix, prefix) + "/")
+}
```

Comment:
I'd remove the `With` from this method and `ListWithPrefix` I don't think the `With` is adding a ton of clarity.

Reply:
Sounds good :+1: